lib/vagrant-proxyconf/action/configure_chef_proxy.rb

```diff
@@ -76,13 +70,24 @@ def configure_chef(chef)
         # @param scheme [String] the http protocol (http or https)
         # @param uri [String] the URI with optional userinfo
         def configure_chef_proxy(chef, scheme, uri)
-          if uri && !chef.public_send("#{scheme}_proxy")
+          if uri && !chef.public_send("#{scheme}_proxy") and !disabled?
```

Comment:
`&&` instead of `and` is more Ruby style

Reply:
Opps, my bad, I'll fix this.  Thanks!

Reply:
Resolved via: https://github.com/tmatilai/vagrant-proxyconf/pull/176/commits/52129592108587a2f0af591af31e118deef4832f